acceptance-tests/features/borrower_views_deed.feature

```diff
@@ -0,0 +1,24 @@
+@borrower_views_deed
+
+Feature: Borrower Views the Deed
+    As a Borrower
+    I want to search for my mortgage document
+    So that I can view it
+
+Background:
+    Given I navigate to the deed search page
+
+Scenario: Borrower enters a valid mortgage reference number
```

Comment:
Lets talk about this scenario, considering you just want to view a deed, all these steps could be removed.